src/spritekit.cs

```diff
@@ -144,14 +144,17 @@ interface SK3DNode {
 
 	[DisableDefaultCtor] // DesignatedInitializer below
 #if MONOMAC
+	[NoiOS][NoTV][NoWatch][NoMacCatalyst]
 	[Mac (10,9)]
 	[BaseType (typeof (NSResponder))]
 	partial interface SKNode : NSSecureCoding, NSCopying {
 #elif IOS || TVOS
+	[NoMac][NoWatch][MacCatalyst (13,0)]
 	[iOS (7,0)]
 	[BaseType (typeof (UIResponder))]
 	partial interface SKNode : NSSecureCoding, NSCopying, UIFocusItem {
 #else // WATCHOS
+	[NoMac][NoiOS][NoTV][NoMacCatalyst]
```

Comment:
After adding this line (specifically the [NoiOS]), xtro complained about the 'AttributeValues', 'GetValue', and 'SetValue' being deprecated for the watch. Online docs show the same so I added the deprecated message